src/Components/Item.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+// import { propTypes } from 'react-bootstrap/esm/Image';
+import { Link } from 'react-router-dom';
+import './item.css';
+
+const Item = (props) => {
+  const {
+    id, title, price, image,
+  } = props;
+  // const {searchItem, setSearchItem}
+
+  const UNITPRICE = 'USD';
+  return (
+    <div className="products">
+
+      <div key={id}>
+        <h3 className="title">{ title }</h3>
+        <div className="price">
+          {' '}
+          {`${UNITPRICE} ${price}`}
+          {' '}
+        </div>
+        <img className="images" src={image} alt="Nice cool products" style={{ width: '200px', height: '200px' }} />
+        <Link to={`/products/${id}`}>
+          <div>see project Details</div>
+        </Link>
+      </div>
+    </div>
+  );
+};
```

Comment:
- [ ] The details page needs some style improvements together with responsiveness to the mobile screens. Currently this is how it looks 👇 
![image](https://user-images.githubusercontent.com/90222110/189453520-f9e2aa56-5dc1-44de-9284-9e735ef03d93.png)